app/Http/Middleware/AcceptContentType.php

```diff
@@ -37,7 +37,7 @@ class AcceptContentType
 	public function handle(Request $request, \Closure $next, string $contentType): mixed
 	{
 		if ($contentType === self::JSON) {
-			if (!$request->expectsJson()) {
+			if (!$request->acceptsJson()) {
```

Comment:
Yes, this is one of the changes I don't like at all. I had my reasons to use `expectsJson`, because otherwise we run into problems with the exception handler. I'm too busy to explain all the details right now, because I am still struggling with the PHPstan PR. I would really be glad, if we could wait with that PR until we have merged @ildyria's PR and maybe my PR about the file streams.